iridauploader/api/api_calls.py

```diff
@@ -663,7 +631,7 @@ def send_sample(self, sample, project_id):
 
         :param sample: Sample object to send
         :param project_id: id of project to send sample too
-        :return: json response from server
+        :return: (int) sample identifier
```

Comment:
This comment needs to be changed back as it is returning a json response and not a sample identifier.

Reply:
Good catch, fixed.